random_menu.rb

```diff
@@ -0,0 +1,46 @@
+# array of food descriptors
+descriptor = [
+  "hot",
+  "crispy",
+  "chilled",
+  "spicy",
+  "sour",
+  "cheesy",
+  "fresh",
+  "vegetarian",
+  "aged",
+  "pickled"
+]
+
+# array of cooking styles
+cooking_style = [
+  "steamed",
+  "fried",
+  "stir-fried",
+  "seared",
+  "sous-vide",
+  "terrine of",
+  "baked",
+  "boiled",
+  "scrambled",
+  "blended",
+]
+
+# array of foods
+food = [
+  "salmon",
+  "chicken breast",
+  "lamb",
+  "cous-cous",
+  "pork",
+  "seafood",
+  "shrimp",
+  "sirloin",
+  "pudding",
+  "omelet"
+]
+
+# random menu generator loop numbering menu items 1 - 10 and taking a random element from descriptor, cooking_style, and foods arrays
+(1..10).each {
+  |n| puts "#{n}. #{descriptor.sample} #{cooking_style.sample} #{food.sample}"
+}
```

Comment:
Style nitpick: we'd like to encourage you to use `do...end` rather than curly braces, and the iteration variable (`n`) should go on the first line (think of it like a method parameter). So this loop would be rewritten as:

```ruby
(1..10).each do |n|
  puts "#{n}. #{descriptor.sample} #{cooking_style.sample} #{food.sample}"
end
```

BTW, using `(1..10).each` is a clever way to get around `10.times` starting at zero. Nice work.